assets/AppleTV/Info.plist

```diff
@@ -7,18 +7,18 @@
 	<key>CFBundleExecutable</key>
 	<string>openssl</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.openssl.OpenSSL</string>
+	<string>com.cossacklabs.OpenSSL</string>
```

Comment:
🤔

Reply:
Well, it's our build. It is not official in any capacity, so I believe we should not use the `org.openssl` namespace.